src/lib/hooks/useFrames.svelte.ts

```diff
@@ -31,7 +31,7 @@ export const provideFrames = (partID: () => string) => {
 	const machineStatus = useMachineStatus(partID)
 	const logs = useLogs()
 	const query = createRobotQuery(client, 'frameSystemConfig', () => ({
-		enabled: environment.current.viewerMode === 'monitor',
+		enabled: partID() !== '' && environment.current.viewerMode === 'monitor',
```

Comment:
Drive by fix that removes a meaningless error that logs if no connected robot is present